{"name": "test_login_params[data_for_login4]", "status": "passed", "attachments": [{"name": "log", "source": "d48f3367-2c2b-44d6-84d5-8dea2dbe212e-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data_for_login", "value": "('', '', 'Fail')"}], "start": 1743589942564, "stop": 1743589945941, "uuid": "f9f081c4-49e8-4dab-a790-3e4d76c3a0d1", "historyId": "5cc684edec223d1983922da115a14b9d", "testCaseId": "5fe6fe3249a01c883d8efece63f2c386", "fullName": "test_login_params.TestLoginParams#test_login_params", "labels": [{"name": "severity", "value": "critical"}, {"name": "tag", "value": "regression"}, {"name": "tag", "value": "sanity"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup', 'log_on_failure')"}, {"name": "suite", "value": "test_login_params"}, {"name": "subSuite", "value": "TestLoginParams"}, {"name": "host", "value": "lukesh"}, {"name": "thread", "value": "18220-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_login_params"}]}